hack/deploy-operator-on-ocp.sh

```diff
@@ -33,17 +33,21 @@ echo "Deploying Operator $OPERATOR_NAME"
 source $SCRIPT_DIR/crds-utils.sh
 apply_crds "deploy"
 
+#Namespace
+oc apply -f deploy/namespace.yaml
+
 # Service Account
-oc create -f deploy/service_account.yaml
+oc apply -f deploy/service_account.yaml
 
 # Roles
-oc create -f deploy/role.yaml
-oc create -f deploy/role_binding.yaml
+oc apply -f deploy/clusterrole.yaml
+oc apply -f deploy/clusterrole_binding.yaml
 
 # Operator
 sed "s/name: kogito-operator/name: $OPERATOR_NAME/g" deploy/operator.yaml > $TMP_FOLDER/operator.yaml
 sed "s|image: quay.io/kiegroup/.*|image: $IMAGE|g" $TMP_FOLDER/operator.yaml > $TMP_FOLDER/operator-tmp.yaml
-oc create -f $TMP_FOLDER/operator-tmp.yaml
+sed '4 a \ \ namespace: operators' $TMP_FOLDER/operator-tmp.yaml > $TMP_FOLDER/operator.yaml
```

Comment:
What is the purpose of this line? And why there is `operators` namespace specified here while other resources like service account have namespace `kogito-operator-system`?

Reply:
Ah yes, this needs to changed to `kogito-operator-system` as well. This line puts `namespace` field in our operator.yaml before applying it, since we want the operator to be installed in that namespace only, it is a hack script to deploy operator with yaml, during testing.

Reply:
changed